haxepunk/Tween.hx

```diff
@@ -61,21 +69,18 @@ class Tween extends EventDispatcher
 		_type = type == null ? TweenType.Persist : type;
 		_ease = ease;
 		_t = 0;
-		_callback = complete;
-		super();
-
-		if (_callback != null)
-		{
-			addEventListener(TweenEvent.FINISH, _callback);
-		}
 	}
 
+	/** @private Update function for override in subclasses */
+	function _update() {}
```

Comment:
I'd love to figure out a way to get rid of that override completely. My goal was to move the finish() code into Tween instead of being called externally by Tweener. The problem was that finish() was being called before the subclass update would run.

I almost called the signal onUpdate but wanted to match what was already there. Seems like a pretty common practice to use a prefix.